src/app/ubs/ubs-admin/components/ubs-admin-table/ubs-admin-table.component.ts

```diff
@@ -136,12 +137,15 @@ export class UbsAdminTableComponent implements OnInit, AfterViewChecked, OnDestr
     private cdr: ChangeDetectorRef,
     private renderer: Renderer2,
     private fb: FormBuilder,
-    private dateAdapter: DateAdapter<Date>
+    private dateAdapter: DateAdapter<Date>,
```

Comment:
[Member 'dateAdapter: DateAdapter<Date>' is never reassigned; mark it as `readonly`.](https://sonarcloud.io/project/issues?pullRequest=3407&open=AZKVdKK5hY9BiVF5AhU8&id=ita-social-projects-green-city-client)